modules/mono/editor/godotsharp_builds.cpp

```diff
@@ -366,6 +367,21 @@ bool GodotSharpBuilds::build_project_blocking(const String &p_config) {
 	pr.step("Building project solution", 0);
 
 	MonoBuildInfo build_info(GodotSharpDirs::get_project_sln_path(), p_config);
+
+	// Add Godot defines
+	String constants = "GodotDefineConstants=\"";
+
+	for (int i = 0; i < p_defines.size(); i++) {
+		constants += "GODOT_" + p_defines[i].to_upper().replace("-", "_").replace(" ", "_").replace(";", " ") + ";";
```

Comment:
Now "my;custom" becomes "MY CUSTOM". Guess underscore/removing makes more sense than a space.

Reply:
Yikes... should have been underscore. My bad!